test/controllers/issues_controller_test.rb

```diff
@@ -18,4 +18,57 @@ class IssuesControllerTest < ActionDispatch::IntegrationTest
     assert_equal test_line_id, issue.line_onestop_id
     assert_response :success
   end
+
+  test "does not allow for visitor to create issue" do
+    test_stop_id = "1"
+    test_line_id = "1"
+    get new_issue_url, params: { stop_id: test_stop_id, line_id: test_line_id }
+
+    assert_response :found
```

Comment:
What is this response assertion? In the [Ruby API docs](https://apidock.com/rails/ActionController/Assertions/ResponseAssertions/assert_response) I only can see `:success`, `:redirect`, `:missing`, and `:error`.